system/metrics/sinks/elastic/elasticSink.cpp

```diff
@@ -38,6 +37,10 @@ ElasticMetricSink::ElasticMetricSink(const char *name, const IPropertyTree *pSet
         configurationValid = true;
         PROGLOG("ElasticMetricSink: Loaded and configured");
     }
+    else
+    {
+        PROGLOG("ElasticMetricSink: Unable to complete initialization, sink not collecting");
```

Comment:
This seems like a warning rather than just progress log. Also, if we know why the problem occurred (either host or index config invalid/missing) we should report it.